mandatory/2-function-creation.js

```diff
@@ -15,8 +21,17 @@ Write a function that:
 - the number must be rounded to 2 decimal places
 - numbers greater 100 must be replaced with 100
 */
-
-function formatPercentage(num) {}
+function formatPercentage(num) {
+    if (num > 100) {
+        return "100%";
+    }
+    if (Number.isInteger(num)) {
+        return `${num}%`;
+    }
+    if (!Number.isInteger(num)) {
+        return `${String(num.toFixed(2).replace(/(\.0+|0+)$/, ""))}%`;
```

Comment:
This works well. There are a few different ways of doing this, so if you have time, look at some of your fellow students' work to see if they have done it differently.